woocommerce/woocommerce-ios
# [Blaze] Intro view image and copy update

Closes: #11578


## Description

The design has been revised after discussions about localizing the "Starting from $5" string displayed on the illustration image. 
 
The price has been removed from the illustration image and moved inside the features text. 
Internal - mBhyoV7OPhFnOugxLy6SMS-fi-2307:50487#656344428

**Changes**
- Updates Blaze campaign creation illustration image
- Update feature copy and rearrange feature order. 


## Testing instructions

- Create a JN site
- Create a new product and publish it
- From the Dashboard Blaze view, tap the "Create Campaign" button
- You should see the new intro view
- Validate that the illustration image and features match the latest design

## Screenshots
| Light | Dark |
|--------|--------|
| ![Simulator Screen Recording - iPhone 14 Pro - 2023-12-29 at 19 18 39](https://github.com/woocommerce/woocommerce-ios/assets/524475/ae1b5fd4-cc91-4e5b-b2ab-62cfec45b6cf) | ![Simulator Screen Recording - iPhone 14 Pro - 2023-12-29 at 19 18 58](https://github.com/woocommerce/woocommerce-ios/assets/524475/f5d01983-a01c-4776-ab43-df29f1c4b0ff) | 

---
